From 3de45274d6a9210e3978a5b2db8dacff34beccb3 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Mon, 5 Aug 2019 23:13:57 +0200
Subject: [PATCH] subversion: update to version 1.10.6

Most interesting changes:
- Fixes CVEs
1.10.6
- CVE-2018-11782
- CVE-2019-0203
1.10.4
- CVE-2018-11803

Other misc changes:
- Fixes indentation in Makefile
- Reorder some things in Makefile to be sync with other packages
- Use HTTPS in URL

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 net/subversion/Makefile | 21 +++++++++++----------
 1 file changed, 11 insertions(+), 10 deletions(-)

diff --git a/net/subversion/Makefile b/net/subversion/Makefile
index 68cae5d01..a169c4df5 100644
--- a/net/subversion/Makefile
+++ b/net/subversion/Makefile
@@ -8,14 +8,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=subversion
+PKG_VERSION:=1.10.6
 PKG_RELEASE:=1
-PKG_VERSION:=1.10.0
+
 PKG_SOURCE_URL:=@APACHE/subversion
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-PKG_HASH:=2cf23f3abb837dea0585a6b0ebd70e80e01f95bddef7c1aa097c18e3eaa6b584
+PKG_HASH:=18c8e72691e6d2e7d3137ae4ad98b9f8977cca5eb2504fab48d55e584b10eca7
+
+PKG_MAINTAINER:=Val Kulkov <val.kulkov@gmail.com>
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE
-PKG_MAINTAINER:=Val Kulkov <val.kulkov@gmail.com>
 
 PKG_FIXUP:=autoreconf
 PKG_MACRO_PATHS:=build/ac-macros
@@ -23,7 +25,6 @@ PKG_MACRO_PATHS:=build/ac-macros
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/nls.mk
 
-
 define Package/subversion/Default
   SECTION:=net
   CATEGORY:=Network
@@ -31,15 +32,15 @@ define Package/subversion/Default
   TITLE:=A compelling replacement for CVS
   DEPENDS:=+zlib +libsqlite3 +PACKAGE_unixodbc:unixodbc +libapr +libaprutil +libmagic \
   	$(ICONV_DEPENDS) $(INTL_DEPENDS)
-  URL:=http://subversion.apache.org/
+  URL:=https://subversion.apache.org/
 endef
 
 define Package/subversion/Default/description
-	Subversion is a free/open-source version control system. That is,
-	Subversion manages files and directories, and the changes made to them,
-	over time. This allows you to recover older versions of your data, or
-	examine the history of how your data changed. In this regard, many
-	people think of a version control system as a sort of time machine.
+  Subversion is a free/open-source version control system. That is,
+  Subversion manages files and directories, and the changes made to them,
+  over time. This allows you to recover older versions of your data, or
+  examine the history of how your data changed. In this regard, many
+  people think of a version control system as a sort of time machine.
 endef
 
 define Package/subversion-libs
-- 
2.17.0.windows.1

